packages/webcert/src/feature/certificate/Inputs/UeRadioGroup.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { CertificateDataElement, QuestionValidationTexts } from '@frontend/common'
+import { useSelector } from 'react-redux'
+import { getQuestionHasValidationError, getShowValidationErrors } from '../../../store/certificate/certificateSelectors'
+import { ConfigTypes, ConfigUeRadioMultipleCodes, RadioButton, ValueCode, ValueCodeList } from '@frontend/common/src'
+import { updateCertificateDataElement } from '../../../store/certificate/certificateActions'
+import { useAppDispatch } from '../../../store/store'
+
+interface Props {
+  disabled: boolean
+  question: CertificateDataElement
+}
+
+const UeRadioGroup: React.FC<Props> = ({ question, disabled }) => {
+  const radiobuttons = (question.config as ConfigUeRadioMultipleCodes).list
+  const isShowValidationError = useSelector(getShowValidationErrors)
+  const shouldDisplayValidationError = useSelector(getQuestionHasValidationError(question.id))
+  const dispatch = useAppDispatch()
+
+  const handleChange: React.ChangeEventHandler<HTMLInputElement> = (event) => {
+    let updatedValue = question
+    updatedValue = getUpdatedValue(question, event.currentTarget.value)
```

Comment:
Why first declare with let and set initial value that will always be overwritten. Use const instead?